requirements-dev.txt

```diff
@@ -2,8 +2,8 @@ behave==1.2.6
 behaving==1.5.6
 flake8==3.7.7
 nose==1.3.7
--e git+https://github.com/qld-gov-au/ckanext-datarequests@develop#egg=ckanext-datarequests
+-e git+https://github.com/qld-gov-au/ckanext-datarequests@training#egg=ckanext-datarequests
```

Comment:
This shouldn't go into the `develop` branch.

Reply:
@MarkCalvert - you may be best to comment on this, but I believe we're using the `training` branch in the test container because there are tests in here that have a dependency on items that have not yet been merged into the `develop` branch for this extension?

Reply:
Yes, this is correct. When working on certain development features that have dependencies from other extensions, the tests will fail until the extension dependencies have been merged into develop. Our current branch workflow is feature -> training -> develop so there for all changes in training end up in develop. But the PR into develop can take weeks which makes the BDD tests fail and therefore unusable.